config/dagu/scripts/git-maintenance.py

```
@@ -0,0 +1,259 @@
+    if not last_success:
+        return True  # Never run before — always due
+    assigned = repo_weekday(repo_name)
+    if today_weekday != assigned:
+        return False
+    elapsed = current_time - parse_iso(last_success)
+    return elapsed >= timedelta(days=GC_INTERVAL_DAYS)
+
+
+def run_gc(repo: Path) -> tuple[bool, str]:
+    result = subprocess.run(
+        ["git", "gc", "--aggressive"],
```

Ruff (0.15.7)

[error] Starting a process with a partial executable path

(S607)